pregred-gui/src/app/components/app-footer/AppFooter.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './AppFooter.css';
+import {
+  FacebookOutlined,
+  InstagramOutlined,
+  YoutubeOutlined,
+  TwitterOutlined
+} from '@ant-design/icons';
+
+
+
+
+function AppFooter() {
+    return(
+      <div className="footer-container">
+        <div className="f-text">
```

Comment:
be more specific on CSS class names. 'dot' and 'f-text' are not understandable. if it is footer text, name that as footer-text.